test/e2e/cypress/support/commands.js

```diff
@@ -64,4 +64,24 @@ Cypress.Commands.add('loadScenario', (scenario) => {
     ]
     cy.log(`Loading scenario "${scenario}"...`)
     cy.exec(`cd ${fixturesPath} && ${photofinishBinary} run --url "http://${collectorHost}:${collectorPort}/api/collect" ${scenario}`)
-})
\ No newline at end of file
+})
+
+Cypress.Commands.add('loadChecksCatalog', (catalog) => {
+    const [
+        fixturesPath,
+        webApiHost,
+        webApiPort
+    ] = [
+        Cypress.env('fixtures_path'),
+        Cypress.env('web_api_host'),
+        Cypress.env('web_api_port')
+    ]
+    cy.log(`Loading checks catalog "${catalog}"...`)
+    cy.fixture(`${catalog}`).then((file) => {
```

Comment:
No need to interpolate, right? `cy.fixture(catalog)` should be enough :thinking: